Code/GraphMol/MolStandardize/TautomerCatalog/TautomerCatalogUtils.cpp

```diff
@@ -34,10 +33,12 @@ MolStandardize::TautomerTransform* getTautomer(const std::string& name,
     throw ValueErrorException("cannot parse tautomer SMARTS: " + smarts);
   }
   tautomer->setProp(common_properties::_Name, name);
-  return new MolStandardize::TautomerTransform(tautomer, bond_types, charges);
+  return std::unique_ptr<MolStandardize::TautomerTransform>(
+      new MolStandardize::TautomerTransform(tautomer, bond_types, charges));
```

Comment:
Now C++14 is supported
```suggestion
  return std::make_unique<MolStandardize::TautomerTransform>(
      tautomer, bond_types, charges);
```

Reply:
good point. Thanks!